src/shell/shell_cmds.cpp

```diff
@@ -496,6 +497,49 @@ static std::vector<int> map_name_lengths(const std::vector<DtaResult> &results,
 	return ws;
 }
 
+static std::vector<int> calc_column_widths(const std::vector<int> &cell_widths,
+                                           int min_cell_width)
+{
+	assert(min_cell_width > 0);
+
+	// Actual text columns using current text mode;
+	// in practice it's either 40, 80, or 132.
+	const int tcols = real_readw(BIOSMEM_SEG, BIOSMEM_NB_COLS);
```

Comment:
The 't' in `tcols` tells me that we need to distinguish whatever these columns are versus some other type of columns: `ncols`, `scols`, etc.  But then I look further and don't see any other `*cols` named variables.. 

The comment tells me that this is actually just the screen-width in characters, and aren't really "columns" of text. Suggest a name that describes it as screen width in characters.